src/Core/3DTiles/C3DTFeature.js

```diff
@@ -1,5 +1,6 @@
 // eslint-disable-next-line no-unused-vars
 import { Object3D, Box3 } from 'three';
+import findBatchTable from './utils/3DTilesUtils';
```

Comment:
`findBatchTable` was moved from `C3DTilesLayer` to its own file. However, it is only used once and not exported to the end-user. Shall we move it here instead?